boundary/assets/dashboards/boundary_overview.json

```diff
@@ -1319,8 +1320,7 @@
     ],
     "template_variables": [],
     "layout_type": "ordered",
-    "is_read_only": false,
     "notify_list": [],
     "reflow_type": "fixed",
-    "id": "83i-2r7-egb"
```

Comment:
Small nit but could we keep this the same as the original?

Reply:
good point! i'll remove those two changes